composer.json

```diff
@@ -25,7 +25,8 @@
     "require": {
         "php": "^7.4.3 || ^8.0 || ^8.1 || ^8.2",
         "ext-curl": "*",
-        "codeigniter4/shield": "^1.0"
+        "codeigniter4/shield": "^1.0",
+        "codeigniter4/settings": "^2.2"
```

Comment:
I’d like to clarify that the Setting package is required for Shield to function properly, and in turn, Shield is essential for Shield OAuth. Therefore, it seems there’s no need for Setting package at this point, as the necessary dependencies are already in place.
```suggestion
```